src/main-process/parse-command-line.js

```diff
@@ -125,6 +128,7 @@ module.exports = function parseCommandLine (processArgs) {
     }
   }
 
+  // Check to see if atomproject flag is set, then add all paths from the .atom-project.
```

Comment:
"atomproject" -> "project", ".atom-project" -> ".atomproject"

hm. now that i think about it, i guess the code doesn't care what the extension is…we should probably pick something (i vote for ".atomproject.json"), be consistent, and make sure that that's advertised as "the" extension in the docstrings and stuff.

Reply:
I think it would be ok to be extension-agnostic now. Why not open it up to any JSON or CSON file?

Reply:
i don't think the code has to care, but imo it's better to have a standard so that we can build tooling on top of it (find all atom project files, etc)